docs/runbooks/release.md

```
@@ -0,0 +1,106 @@
+
+```
+LOG=$(mktemp)
+make build 2>&1 | tee -a "$LOG"
+make test  2>&1 | tee -a "$LOG"
+```
+
+## 4) Write the Journal Entry
+
+Set parameters via environment (non‑interactive, CI‑friendly) and attach logs via `SHIPLOG_LOG`.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)